### PR TITLE
created deploy script + changed target server + moved key

### PR DESCRIPTION
**Pull request by LosDrakakos**
LB will need to have their conf updated after deployment.

La bise

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,14 +26,14 @@ script:
...
+  - openssl aes-256-cbc -K $encrypted_5b0d89742976_key -iv $encrypted_5b0d89742976_iv -in deploy/deploy_rsa.enc -out /tmp/deploy_rsa -d
   - eval "$(ssh-agent -s)"
   - chmod 600 /tmp/deploy_rsa
   - ssh-add /tmp/deploy_rsa
 
 deploy:
   provider: script
   skip_cleanup: true
-  script: rsync -r --delete-after --quiet $TRAVIS_BUILD_DIR automate@back1.igotta.beer:/var/www/geekdpt.io && killall php && cd /var/www/geekdpt.io/gklandingpage && nohup php server.php &
+  script: /bin/bash /var/www/html/geekdpt.io/gklandingpage/deploy/travis_deploy.sh $TRAVIS_BUILD_DIR
```
Comment (Neofox) [inline .travis.yml]: You should replace that by `bash deploy/travis_deploy.sh $TRAVIS_BUILD_DIR`

File: deploy/travis_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+else
+  echo "Vars are OK, let's go"
+fi
+if rsync -r --delete-after --quiet $TRAVIS_BUILD_DIR automate@back1.igotta.beer:/var/www/geekdpt.io; then
+  echo "rsync OK"
+else
+  echo "rsync failed"
+  exit 1
+fi
+if killall php; then
```
Comment (Neofox) [inline deploy/travis_deploy.sh]: This command should be done on the server, not in the travis environment

File: deploy/travis_deploy.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  echo "all php instances were killed \n Oh you mass murderer..."
+else
+  echo "I'm sorry, I couldn't kill them... They have feelings y'know!"
+  exit 1
+fi
+if cd /var/www/geekdpt.io/gklandingpage; then
```
Comment (Neofox) [inline deploy/travis_deploy.sh]: this one too

File: deploy/travis_deploy.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  echo "I'm in position commander, awaiting orders"
+else
+  echo "I'm the way is obstructe commander, aborting mission"
+  exit 1
+fi
+if screen -d -m php server.php; then
```
Comment (Neofox) [inline deploy/travis_deploy.sh]: and this one

**Review by Neofox — CHANGES_REQUESTED** [review]

**Comment by LosDrakakos** [comment]
fixed

**Review by Neofox — APPROVED** [review]

**Pull request merged**